meson_options.txt

```diff
@@ -23,5 +23,4 @@ option('build_subprojects', type : 'boolean', value : true, yield : true)
 
 option('enable-ysnp', type: 'boolean', value: true)
 
-option('manpages', type: 'boolean', value: true, description: 'Install manpages for all CLI tools/examples/tests')
-option('bash_completion', type: 'feature', value: 'disabled', description: 'Install bash-completions for all CLI tools/examples/tests')
+option('force_completions', type: 'boolean', value: false, description: 'Install bash-completions at the default location if bash-completion is not available on the system')
```

Comment:
Perhaps rephrase as:

``Install bash-completions at the default location even when Bash is not available on the system``

Also, you remove two build-options and add another one. Thus, this change is closer to a refactoring and it is a build-system-API-breaking one. Thus, the commit message should signify that with:

``refactor(meson)!: replace options and only install manpages/completions for installed binaries``

Reply:
> Install bash-completions at the default location even when Bash is not available on the system

On Arch linux Bash does not include `bash-completion`, they are separate packages.
In the `meson.build` we check for `bash-completion`

Reply:
> Also, you remove two build-options and add another one. 

I will edit this, so removing the man-pages option goes into the correct commit